README.md

```diff
@@ -41,13 +41,65 @@ $runner->addExtension(new \mageekguy\atoum\ruler\extension($script));
 
 ## Use it
 
-Examples :
+The extension adds a `--filter` option to atoum. This line now appears on the atoum help :
+
+```
+--filter: Filters tests to execute. For example 'not("featureA" in tags) and namespace = "foo\bar"'
+```
+
+You can now filter your tests using any [\Hoa\Ruler filter](https://github.com/hoaproject/Ruler).
+
+Those variables are available in the filter :
+* method
+* class
+* namespace
+* testedclass
+* testedclassnamespace
+* tags (as an array)
+
+## Examples
+
+Run all tests except those who have the "needsDatabase" tag :
+```
+./vendor/bin/atoum -d tests --filter 'not ("needsDatabase" in tags)'
+```
+
+Run all tests with a method named "testMethod1" :
 ```
-./vendor/bin/atoum -d tests --filter 'not ("unSuperTagAuNiveauDeLaMethode1" in tags)'
 ./vendor/bin/atoum -d tests --filter 'method = "testMethod1'
+```
+
+
+Run all tests with a method named "testMethod1" (using an an array to contain the list of filtered methods) :
+```
 ./vendor/bin/atoum -d tests --filter 'method in ["testMethod1"]'
+```
+
+
+You can also defined more complex filters like this : run all tests tagged tagged "needsDatabase" and with method "testClass1" or with a method "testClass" :
```

Comment:
First word after “:” in English is like the first word of a sentence, so with its first letter in uppercase. Example here: “You can also define more complex filter such as: Run all tests tagger”…

Also, two times “tagged” here :-).